model/flow/seal_test.go

```diff
@@ -0,0 +1,26 @@
+package flow_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+// Test_SealID checks that two seals that only differ in their approval
+// signatures have different IDs. This is REQUIRED FOR the STORAGE layer
+// to correctly retrieve the block payload!
+func Test_SealID(t *testing.T) {
+
+	seal := unittest.Seal.Fixture()
+	id := seal.ID()
+	cs := seal.Checksum()
+
+	// Change signatures of first chunk
+	seal.AggregatedApprovalSigs[0] = unittest.Seal.AggregatedSignatureFixture()
+
+	// They should not have changed
```

Comment:
```suggestion
	// They should be different
```

We're asserting that they aren't equal here, so they should have changed